src/vivarium/framework/values.py

```diff
@@ -111,10 +111,15 @@ def rescale_post_processor(value: NumberLike, manager: "ValuesManager") -> Numbe
 
     """
     if hasattr(value, "index"):
-        time_step = manager.simulant_step_sizes(value.index).dt
+        return value.mul(
```

Comment:
The mortality pipeline is a dataframe, and unfortunately, the default behavior of `mul` is to multiply  dataframe x series across the columns, not the index.